tox.ini

```diff
@@ -405,7 +405,7 @@ skip_install = True
 deps =
     tomte[safety]==0.1.5
 commands =
-    safety check -i 37524 -i 38038 -i 37776 -i 38039 -i 39621 -i 40291 -i 39706 -i 41002
+    safety check -i 37524 -i 38038 -i 37776 -i 38039 -i 39621 -i 40291 -i 39706 -i 41002 -i 51499 -i 51358
```

Comment:
@DavidMinarsch safety check is failing with 

```
+============================+===========+==========================+==========+
| safety                     | 1.10.3    | <2.2.0                   | 51358    |
| wheel                      | 0.37.1    | <0.38.0                  | 51499    |
+==============================================================================+
```

I think we'll have to bump safety in the next tomte release